projects/addon-editor/components/editor-socket/editor-socket.component.ts

```diff
@@ -31,7 +31,10 @@ export class TuiEditorSocketComponent {
             this.elementRef.nativeElement,
             'innerHTML',
             this.tuiSanitizer
-                ? this.tuiSanitizer.sanitize(SecurityContext.HTML, content)
+                ? this.tuiSanitizer.sanitize(
+                      SecurityContext.HTML,
+                      content.replace(/colwidth/g, 'width'),
```

Comment:
This is a temporary workaround.  Tables extension (with  'resizable': true option} generate attribute 'colwith' for columns instead of 'width'.  https://github.com/ueberdosis/tiptap/issues/1766
I just wanted to discuss to find a better way